cmd/catalog.go

```diff
@@ -38,14 +38,32 @@ var catalogListCmd = &cobra.Command{
 		case 0:
 			fmt.Printf("No deployable components found\n")
 		default:
-
+			currentProject := client.GetCurrentProjectName()
 			w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
-			fmt.Fprintln(w, "NAME", "\t", "TAGS")
+			fmt.Fprintln(w, "NAME", "\t", "NAMESPACE", "\t", "TAGS")
```

Comment:
we should use `PROJECT` instead of `NAMESPACE`, user may get confused since we have `odo project xxxx` as well

Reply:
I have changed in user appearance areas. It seems like we have maintained it as namespace internally in code in a good number of places..